packages/fiori/src/Interfaces.ts

```diff
@@ -1,65 +1,47 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import type { ITabbable } from "@ui5/webcomponents-base/dist/delegate/ItemNavigation.js";
+import TimelineLayout from "./types/TimelineLayout.js";
+
 /**
  * Interface for components that may be slotted inside <code>ui5-page</code> as header and footer.
```

Comment:
Not only "ui5-page"

Reply:
Need to think about other description, because the interface is currently used only in the page's documentation since the component is placed in fiori package.